src/object_detector/operations.jl

```diff
@@ -0,0 +1,229 @@
+import Statistics
+import Images
+
+
+
+""""
+nanmean()
+    computing the mean of an array, x, while neglegting NaN values.
+"""
+nanmean(x) = Statistics.mean(filter(!isnan,x))
```

Comment:
The nan functions are not really needed anymore. They should probably just be removed.